src/status_im/ui/screens/syncing/sheets/enter_password/views.cljs

```diff
@@ -0,0 +1,88 @@
+(ns status-im.ui.screens.syncing.sheets.enter-password.views
+  (:require [quo.react-native :as rn]
+            [status-im.ui.screens.syncing.sheets.enter-password.styles :as styles]
+            [re-frame.core :as re-frame]
+            [quo.core :as quo]
+            [status-im.i18n.i18n :as i18n]
+            [status-im.ui.components.react :as react]
+            [status-im.ui.components.toolbar :as toolbar]
+            [clojure.string :as string]
+            [taoensso.timbre :as log]
+            [status-im.utils.security :as security]
+            [status-im.ethereum.core :as ethereum]
+            [status-im.utils.fx :as fx]
+            [status-im.native-module.core :as status]
+            [quo2.components.info.information-box :as information-box]
+            [status-im.ui.components.qr-code-viewer.views :as qr-code-viewer]
+            [status-im.utils.handlers :refer [>evt]]))
+
+(def entered-password (atom ""))
+(def connection-string  (atom ""))
+
+(defn qr-code-view-with-connection-string [connection-string]
+  (log/info "connection-string ===> " connection-string)
+  (let [window-width       @(re-frame/subscribe [:dimensions/window-width])
+        valid-cs?          (string/starts-with? connection-string "cs")]
+    [:<>
+     (if valid-cs?
+       [rn/view {:style styles/body-container}
+        [rn/text {:style styles/header-text} (i18n/label :t/sync-code-generated)]
+        [qr-code-viewer/qr-code-view (* window-width 0.808) connection-string]
+        [information-box/information-box {:type      :informative
+                                          :closable? false
+                                          :icon      :main-icons2/placeholder
+                                          :style     {:margin-top 20}} (i18n/label :t/instruction-after-qr-generated)]]
+       [rn/view {:style styles/body-container}
+        [rn/text {:style styles/header-text} connection-string]
+        [toolbar/toolbar
+         {:size :large
+          :center
+          [react/view {:padding-horizontal 8}
+           [quo/button
+            {:on-press #(re-frame/dispatch [:preperations-for-connection-string])}
+            (i18n/label :t/try-your-luck-again)]]}]])]))
+
+(fx/defn initiate-local-pairing-with-connection-string
+  {:events [:initiate-local-pairing-with-connection-string]}
+  [{:keys [db]} {:keys [data]}]
+  (let [config-map         (.stringify js/JSON (clj->js {:keyUID "" :keystorePath "" :password ""}))
+        connection-string  data]
+    (status/input-connection-string-for-bootstrapping
+     connection-string
+     config-map
+     #(log/info "this is response from initiate-local-pairing-with-connection-string " %))))
+
+(fx/defn preperations-for-connection-string
+  {:events [:preperations-for-connection-string]}
```

Comment:
About the name of the event and the other `:initiate-local-pairing-with-connection-string`:

- We should avoid keywords without qualification in the global re-frame event registry.
- It's confusing to me, something like `:syncing/get-connection-string-for-bootstrapping` is easier to reason.